modules/EvseV2G/v2g_server.cpp

```diff
@@ -162,6 +163,12 @@ static int v2g_incoming_v2gtp(struct v2g_connection* conn) {
         return -1;
     }
 
+    if (conn->payload_len >= UINT32_MAX - V2GTP_HEADER_LENGTH) {
+        dlog(DLOG_LEVEL_ERROR, "Prevent integer overflow - payload too long: have %d, would need %d",
+             DEFAULT_BUFFER_SIZE, conn->payload_len);
```

Comment:
Shouldn't the second parameter be "%u" because payload_len is of type uint32_t?

Reply:
Good point. Then I have to change the next dlog() in line 173, too.

Reply:
Done